typings/jsDocs.ts

```diff
@@ -952,6 +952,19 @@ type Group = {
     isLocked: boolean;
 }
 
+/**
+ * @typedef
+ */
+type GroupSearchItem = {
+    id: number;
+    name: string;
+    description: string;
+    memberCount: number;
+    publicEntryAllowed: boolean;
+    created: Date
```

Comment:
Missing semi-colons.

Reply:
There are a lot of lines in both typings files without semicolons - Is this something required by typescript? Generally in the library we conform to Standardjs, which prohibits semicolons.